dhis-2/dhis-web/dhis-web-api-test/pom.xml

```diff
@@ -1,195 +1,200 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
It would be great to figure out why the entire file is reformatted, this makes it hard to see what _really_ changed and will probably be changed back in a later commit.